### PR TITLE
BIBD notebook tutorial

### PR DESCRIPTION
**Pull request by sasha704**
This is a pull request for the BIBD notebook tutorial.

### CODE REVIEW

**Comment by ChrisJefferson** [comment]
Hi! I'm sorry for not looking at this earlier. It hink it's a great idea!

Is there an easy way for me to run this, to see what it looks like? just looking through the PR it looks good, and I think being able to do this in a live notebook will make it much easier for people to interact with.

**Comment by ozgurakgun** [comment]
Good point! Could you please share the link to colab on the issues @sasha704? I think we want everyone to have read access & Google will let them create a new copy when they access it.

**Comment by sasha704** [comment]
Yes definitely @ozgurakgun. 

@ChrisJefferson [Here's the link for the relevant colab for BIBD](https://colab.research.google.com/drive/1drFmrMoGBW4__rB1wDhFOE3OlgWRHGz9?usp=sharing)

**Comment by ozgurakgun** [comment]
Thanks @sasha704, this is very helpful!

File: docs/tutorials/notebooks/BIBD.ipynb [hunk 1]
```diff
@@ -0,0 +1,912 @@
...
+          },
+          "metadata": {},
+          "execution_count": 4
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "With no constraints it produces an empty set for crop assignment."
```
Comment (ChrisJefferson) [inline docs/tutorials/notebooks/BIBD.ipynb:485]: Maybe "with no constraints Conjure produces an empty set for the crop assignment"?

File: docs/tutorials/notebooks/BIBD.ipynb [hunk 2]
```diff
@@ -0,0 +1,912 @@
...
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{'crop_assignment': [['🥔', '🥦', '🍅'],\n",
+              "  ['🥔', '🥕', '🥒'],\n",
+              "  ['🌽', '🥦', '🥒'],\n",
+              "  ['🌽', '🥕', '🍅']]}"
```
Comment (ChrisJefferson) [inline docs/tutorials/notebooks/BIBD.ipynb:524]: This solution actually does have the correct number of assigned crops!

I re-ran the model and got something different, so maybe just re-run eerything to force a different answer?

**Comment by ChrisJefferson** [comment]
This looks good. The most important thing is to re-run the whole thing I think, to fix that cell whose answer looks wrong (well, actually too right! :) )

**Comment by ozgurakgun** [comment]
I merged the other ones, and they look good! See [here](https://conjure.readthedocs.io/en/latest/tutorials-notebook.html)

Let me know when this one is ready please and we can merge this one too.

**Comment by ozgurakgun** [comment]
hi @sasha704 - wondering if you could take a look at this PR at some point? If you can't, I can look into it, just let me know please.

**Comment by sasha704** [comment]
@ozgurakgun 
Sorry for the slow response, I've just updated this pull request with the re-run version of the notebook. It should be ready to pull now!

**Pull request merged**